src/Git/Value/MergeTargetCandidateBranches.php

```diff
@@ -101,4 +102,41 @@ public function branchToMergeUp(SemVerVersion $version): ?BranchName
             ? null
             : $branch;
     }
+
+    /** @TODO move `master` exclusion into ctor - `master` is no longer a valid release branch */
```

Comment:
This change will likely lead to a BC break. Not a problem, since the library is not yet released, but good to keep in mind that the tool will change massively, and that `feature/` files will need adjusting